nextcord/ext/help_commands/embedded.py

```diff
@@ -0,0 +1,192 @@
+from typing import Mapping, Optional, Union
+
+from nextcord import Embed, Message
+from nextcord.ext import commands
+
+
+class EmbeddedHelpCommand(commands.HelpCommand):
+    """A help command implementation using embeds.
+    This inherits from :class:`HelpCommand`.
+    It extends it with the following attributes.
+
+    Attributes
+    ------------
+    dm_help: :class:`bool`
+        A bool that indicates if the output should only be sent via DM instead of the channel.
+        Defaults to False.
+    default_color: :class:`int`
+        The default color of all embeds.
+        Defaults to 0xFFFFFF (White).
+    main_embed_title: :class:`str`
+        The title for the embed which gets sent by invoking `help`.
+        Defaults to "Overview about Cogs, Groups and Commands".
+    main_embed_description: Optional[:class:`str`]
+        The description for the embed which gets sent by invoking `help`.
+    main_embed_color: Optional[:class:`int`]
+        The color for the embed which gets sent by invoking `help`.
+        Defaults to default_color.
+    command_embed_title: :class:`str`
+        The title for the embed which gets sent by `help <command>`.
+        Defaults to "Command help".
+    command_embed_description: :class:`str`
+        The description for the embed which gets sent by invoking `help <command>`.
+        Defaults to "If a parameter is surrounded by `<>`, it is a required parameter\nIf a parameter is surrounded by `[]`, it is an optional parameter."
```

Comment:
```suggestion
        Defaults to "If a parameter is surrounded by `<>`, it is a required parameter\\\\nIf a parameter is surrounded by `[]`, it is an optional parameter."
```

The backslash needs to be escaped if you want it to show as `\n` in the docs:

![image](https://user-images.githubusercontent.com/20955511/181864036-a22fc6ad-9b6c-4fa3-9c8e-d168f300e20c.png)

Otherwise it looks like this:

![image](https://user-images.githubusercontent.com/20955511/181864049-586cfa41-e9b7-40e5-8658-b8e6a799632b.png)

Reply:
Also maybe there should be a period at the end of the first sentence before the \n to make it consistent with the second.

Reply:
@H4ckerxx44 Looks like you changed this in the wrong spot. The docs need the four backslashes, the actual string in the code should only have one.